src/public/sharing.ts

```diff
@@ -77,19 +77,19 @@ export namespace messaging {
       FrameContexts.meetingStage,
     );
 
-    sendMessageToParent(MessagingAPIMessages.shareWebContent, [shareWebContentRequest], callback);
+    sendMessageToParent(SharingAPIMessages.shareWebContent, [shareWebContentRequest], callback);
   }
 
   /**
    * @private
    * Feature is under development
    *
    * Opens a share dialog for creating a class asignment
-   * @param shareWebContentRequest web content info
+   * @param createAssignmentRequest assignment info
    * @param callback optional callback
    */
   export function createAssignment(
```

Comment:
Is this API needed for v1 of this feature? Is this covered in Pankaj's PM spec? I personally don't think we should force the apps to choose how they want to share the web content because they often won't know what's appropriate. Apps don't even have a way to know whether Assignments is even available (which is only in EDU tenants).

Reply:
I've removed create assignment, deferring it to a future add-on.